molo/core/models.py

```diff
@@ -147,14 +147,7 @@ class BannerPage(TranslatablePageMixin, Page):
 BannerPage.content_panels = [
     FieldPanel('title', classname='full title'),
     ImageChooserPanel('banner'),
-    PageChooserPanel('banner_link_page'),
-    MultiFieldPanel(
-        [
-            FieldPanel('commenting_state'),
-            FieldPanel('commenting_open_time'),
-            FieldPanel('commenting_close_time'),
-        ],
-        heading="Commenting Settings",)
```

Comment:
@justinvdm Commenting Settings has been added by mistake to banner pages, a banner page doesn't have commenting option, basically a user can't comment on a banner, so doesn't need commenting settings